crates/services/executor/src/executor.rs

```diff
@@ -1073,6 +1074,20 @@ where
             })
         }
 
+        // TODO: Move to an off-chain worker: https://github.com/FuelLabs/fuel-core/issues/1721
```

Comment:
Since it is a temporary solution, I'm okay with doing it here. Later, we will move this code to the off-chain worker.